src/SFA.DAS.Assessor.Functions.ExternalApis/Config/OAuth.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.Assessor.Functions.ExternalApis.Config
```

Comment:
this isn't all required now for use with Managed Identities

Reply:
Removed